torchmultimodal/modules/layers/attention.py

```diff
@@ -194,26 +194,18 @@ class FullAttention(nn.Module):
 
```

Comment:
Add docstring for `attention_mask` and `head_mask` with dimensions.
The same for `MultiHeadAttention`.